README.md

```diff
@@ -16,11 +16,7 @@ bun dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```

Comment:
Feel free to modify the README, You can keep a link to the original source setup for the default readme.

## Env Required

## Stack (URL with the version of 3erd party)
## How to Run

## How to Deplouy (i guess this is empty yet)